gammapy/background/energy_offset_array.py

```diff
@@ -3,6 +3,11 @@
 import numpy as np
 from astropy.coordinates import Angle
 from astropy.units import Quantity
+from astropy.table import Table, Column
+from astropy.io import fits
+from scipy import interpolate
```

Comment:
You have to move the import into the `evaluate` method:
https://travis-ci.org/gammapy/gammapy/jobs/108560761#L910

And mark the test for the evaluate method with
```
@requires_dependency('scipy')
```

Example: https://github.com/gammapy/gammapy/blob/master/gammapy/irf/tests/test_energy_dispersion.py#L58